### DISCUSSION TITLE
Wish List: Link to original post in "New comment" emails, "Edit" button on archive pages

### DISCUSSION THREAD

**Discussion by krangor** [buttondown/discussions#30] [Ideas]
As per the title, would be nice to be able to jump to the page directly from the notification email.

Would also be nice to leap to the editor directly from the archive page instead of going to the editor and hunting for it.

**Comment by krangor** [comment]
ALSO: would be nice for replies to sent emails to just be their own email chain without a Buttondown intermediary thing.

**Comment by jmduke** [comment]
Filed here (https://github.com/buttondown/roadmap/issues/2603) and here(https://github.com/buttondown/roadmap/issues/2604)!

> ALSO: would be nice for replies to sent emails to just be their own email chain without a Buttondown intermediary thing.

You _can_ do that if you opt out of tracked replies (though this requires you to be sending from a public email!)

**Reply by krangor** [reply]
Is there a world where it would just track the first reply and then like, de-chaperon itself?

**Reply by krangor** [reply]
I just woke up but I hope that makes sense.
